api/jwt/generate_token.go

```diff
@@ -66,5 +72,10 @@ func GenerateRefreshToken(userId string, atJti string, rtJti string) (store.Toke
 	if err != nil {
 		return td, err
 	}
+
+	errAccess := ts.StoreRefresh(userId, td)
```

Comment:
At first glance, this line looks as if we are storing the whole token detail, which is surprising (since a complete detail is not usually stored). Turns out, we only stores the uuid and exp. I suggest we pass only what is needed, so the StoreRefresh function signature can be 
```
StoreRefresh(userId string, accessUUID string, expires int64) error {
```

similar comment to `StoreAccess` function